backend/src/main/java/com/crm/model/db/CarTypeEntity.java

```diff
@@ -10,9 +9,13 @@
 import javax.persistence.Table;
 
 @Entity
-@Table(name = "car_types")
-@Getter
 @NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
+@Setter
+@EqualsAndHashCode
+@Builder
```

Comment:
To samo

Reply:
To nie zadziała z Builderem, zostawiłem jak było